server/session/handler_enchanting.go

```diff
@@ -0,0 +1,341 @@
+package session
+
+import (
+	"fmt"
+	"github.com/df-mc/dragonfly/server/block"
+	"github.com/df-mc/dragonfly/server/block/cube"
+	"github.com/df-mc/dragonfly/server/internal/sliceutil"
+	"github.com/df-mc/dragonfly/server/item"
+	"github.com/df-mc/dragonfly/server/world"
+	"github.com/sandertv/gophertunnel/minecraft/protocol"
+	"github.com/sandertv/gophertunnel/minecraft/protocol/packet"
+	"golang.org/x/exp/slices"
+	"math"
+	"math/rand"
+)
+
+// enchantNames are names translated to the 'Standard Galactic Alphabet' client-side. The names generally have no meaning
+// on the vanilla server implementation, so we can sneak some easter eggs in here without anyone noticing.
+var enchantNames = []string{
+	"dragonfly",
+	"gophertunnel",
+	"go raknet",
+	"go lang",
+	"sandertv",
+	"t 14 raptor",
+	"da pig guy",
+	"potatoe train yt",
```

Comment:
We might want to create a better list with much more entries, or grab one from the internet. There's a pretty good chance you'll see duplicates with just this.

Reply:
I had the idea of grabbing all contributors at runtime once. Twisted got this call which works: `curl -H "Accept: application/vnd.github+json" https://api.github.com/repos/df-mc/dragonfly/contributors`. This gives a json array, we could just parse that and compile a list by inserting a space after every uppercase letter, then lowercasing the entire thing.

Reply:
Thanks Pig!!!!